app/views/api/v0/webhooks/index.json.jbuilder

```diff
@@ -0,0 +1,8 @@
+json.array! @webhooks.each do |endpoint|
+  json.id             endpoint.id
```

Comment:
let's add the `json.type_of            "webhook"` field just for consistency with the other APIs